src/script/cpatch.py

```diff
@@ -513,39 +529,29 @@ def _copy_binary(self, src_path, dst_path):
             pass
         os.link(src_path, dst_path)
 
-    def _bins_and_libs(self, prefix, bin_patterns, lib_patterns):
+    def _bins_and_libs(self, exclude_prefixes, bin_patterns, lib_patterns):
         out = []
 
         bin_src = self._ctx.build_dir / "bin"
-        bin_dst = self._workdir / f"{prefix}_bin"
```

Comment:
I don't understand why this needs to change. Why isn't prefix just set to "tmp" where appropriate?

Reply:
I changed this because prefix was always being set to "tmp" so it didn't seem to add any value.
All binaries end up in the tmp_bin directory because that is where the Dockerfile build process will search for them when it patches the existing files in the container.  
I can revert this if you would prefer.

Reply:
> I can revert this if you would prefer.

Yes please. Let's make the changes as small and targeted as possible.

Reply:
done